src/components/ConfirmedRoute.js

```diff
@@ -93,6 +93,10 @@ function ConfirmedRoute({mapboxAccessToken, transaction}) {
                     accessToken={mapboxAccessToken.token}
                     mapPadding={CONST.MAP_PADDING}
                     pitchEnabled={false}
+                    initialState={{
+                        zoom: CONST.MAPBOX.DEFAULT_ZOOM,
+                        location: waypointMarkers[0].coordinate,
+                    }}
```

Comment:
```suggestion
                    initialState={
                        _.isEmpty(waypointMarkers)
                            ? undefined
                            : {
                                  zoom: CONST.MAPBOX.DEFAULT_ZOOM,
                                  location: waypointMarkers[0].coordinate,
                              }
                    }
```

Reply:
I just committed with
`location: lodashGet(waypointMarkers, [0, 'coordinate'], CONST.MAPBOX.DEFAULT_COORDINATE),`

Is it looks fine? bcoz we also need to use default one in case waypoint coordinate is not available

Reply:
That makes sense too.
When visit this page, coordinate should always exist as this came from pass of previous step.
As direct deep link is edge case, I think it's fine just to prevent crash here.

Reply:
Do you want me to update from lodash way to pass undefined as suggested?

Reply:
Your code is fine to me